README.md

```diff
@@ -244,14 +242,14 @@ open-attestation decrypt ./src/__tests__/fixture/did-dns-encrypted.json decrypte
 ✔  success   Decrypted document saved to: decrypted.json
 ```
 
-### Token registry
+### Token Registry
 
-#### Deploy new token registry
+#### Deploy new Token Registry
 
-Deploys a token registry contract on the blockchain
+Deploys a token registry contract on the blockchain. Factory Contract that have been deployed using token-registry can be used with the factory address flag. To deploy a standalone token registry, please refer to [Token-Registry](https://github.com/Open-Attestation/token-registry) deployment.
 
```

Comment:
is it correct to say without `--factory-address`, it fallbacks to [these](https://github.com/Open-Attestation/token-registry/blob/70ff40eea7406866fb595147522a964c8d78f3b6/src/constants/contract-address.ts#L10-L16)?